packages/material-ui/src/RadioGroup/RadioGroup.js

```diff
@@ -9,13 +9,14 @@ import { createChainedFunction, find } from '../utils/helpers';
 class RadioGroup extends React.Component {
   radios = [];
 
-  state = {
-    value: null,
-  };
-
   constructor(props) {
-    super();
-    this.isControlled = props.value != null;
+    super(props);
```

Comment:
You shouldn't need to use super(props) here https://github.com/mui-org/material-ui/pull/12476

Reply:
done